src/main/no/nav/pto/veilarbfilter/routes/MineLagredeFilterRoutes.kt

```diff
@@ -81,8 +81,11 @@ fun Route.mineLagredeFilterRoutes(mineLagredeFilterService: FilterService, useAu
     }
 }
 
-private fun ApplicationCall.getNavident(): String? {
+fun ApplicationCall.getNavident(): String? {
+    if (this.principal<JWTPrincipal>()?.payload?.claims?.containsKey("NAVident")!!) {
+        return this.principal<JWTPrincipal>()?.payload?.getClaim("NAVident")?.asString();
```

Comment:
Ville kanskje hentet `this.principal<JWTPrincipal>()?.payload?.getClaim("NAVident")` ut som en variabel og gjort en sjekk på at den ikke er null, så slipper man å hente den ut flere ganger.